### ISSUE TITLE
Parser Error: import uuid = require('uuid');

### ISSUE THREAD

**Issue by useafterfree** [snowplow/snowplow-javascript-tracker#799]
```
  ERROR in ./node_modules/snowplow-tracker-core/lib/core.ts 16:12
  Module parse failed: Unexpected token (16:12)
  You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
  |  */
  | 
  > import uuid = require('uuid');
  | 
  | import * as payload from './payload';
   @ ./node_modules/snowplow-tracker-core/index.ts 16:0-39 16:0-39
   @ ./src/App.tsx
   @ ./src/index.tsx
   @ multi @babel/polyfill ./src/index.tsx webpack-plugin-serve/client
```

**Comment by paulboocock** [comment]
This looks like you are trying to use our core library inside a React application from npm.

The core library is designed as a shared codebase between our node tracker (for server applications), and our JS tracker (for web applications). For tracking in React applications, we advise that you should use the JavaScript tracker by adding the `sp.js` file from our Github releases as an asset to your site, and then load it with our snippet (see here: https://github.com/snowplow/snowplow/wiki/integrating-javascript-tags-onto-your-website#1-setting-up-snowplow-pageview-tracking).

The Snowplow JavaScript tracker is not currently available on NPM but we intend for this to be released later this year. 

If you do with to use snowplow-tracker-core in your react application, you will need to build a wrapper around it for sending the events but as a library it should work fine, I've certainly tested and used it before so I assume this error is something to do with your configuration. If there is another issue somewhere that I'm missing here then I'll need much more detail and perhaps a simple repro of this issue too.

As a simple exercise, I created a new application using https://create-react-app.dev/ and ran `npm install snowplow-tracker-core`. 
I then made the following adjustments to `App.js`:
```
import React from 'react';
import logo from './logo.svg';
import './App.css';
var core = require('snowplow-tracker-core').trackerCore;

function App() { 

  function test() {
      var coreInstance = core(false);
      coreInstance.addPayloadPair('vid', 2);
      coreInstance.addPayloadDict({
          'ds': '1160x620',
          'fp': 4070134789
      });
      coreInstance.setTrackerVersion('js-3.0.0');
      coreInstance.setPlatform('web');
      coreInstance.setUserId('user-321');
      var pageViewPayload = coreInstance.trackPageView('http://www.example.com', 'landing page');
      console.log(pageViewPayload.build());
  }

  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" onClick={() => test()} style={{"pointerEvents": "all"}}/>
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
```

**Comment by paulboocock** [comment]
Will be resolved once #541 is complete.
